### ISSUE TITLE
What is the significance of the order in color translation function?

### ISSUE THREAD

**Issue by kkiwimagi** [TASBE/TASBEFlowAnalytics-Tutorial#6]
What is the significance of the order in color translation function?

% Multi-color controls are used for converting other colors into FITC units
colorpairfiles = {};
% Entries are: channel1, channel2, constitutive channel, filename
% If you only have two colors, you can set consitutive-channel to equal channel1 or channel2
colorpairfiles{1} = {channels{1}, channels{2}, channels{3}, [stem0312 'mkate_EBFP2_EYFP_P3.fcs']};
colorpairfiles{2} = {channels{1}, channels{3}, channels{2}, [stem0312 'mkate_EBFP2_EYFP_P3.fcs']};

In other words, would the following execute the same operation?

colorpairfiles{1} = {channels{3}, channels{2}, channels{1}, [stem0312 'mkate_EBFP2_EYFP_P3.fcs']};
colorpairfiles{2} = {channels{1}, channels{3}, channels{2}, [stem0312 'mkate_EBFP2_EYFP_P3.fcs']};

**Comment by jakebeal** [comment]
The three entries are "Channel 1", "Channel 2", and "Constitutive Channel".  The first two are symmetric, and if the third is different, it's used for binning the first two (which is more independent).
